src/dmd/mtype.d

```diff
@@ -7353,3 +7353,49 @@ const(char)* toChars(ScopeRef sr) pure nothrow @nogc @safe
         return names[sr];
     }
 }
+
+/** Returns: `true` iff `t` has any non-`immutable` (mutable or const) indirections.
```

Comment:
Instead of repeating what the `Returns:` section says, something better would be:

"Aliasing happens when there exists another mutable reference to the same memory object.
The other references can change the contents of the memory object.
Immutable references may exist, but cannot change the contents. This function determines if
such aliasing of a type is possible."

Reply:
Thanks. Done.